test/include.mk

```diff
@@ -153,11 +153,19 @@ test-%.sh.out: $(TEST_DIR)/test-%.sh
 
 test: test-k
 
+test-simplifierx: test-k-simplifierx
+
+test-k-simplifierx: KORE_EXEC_OPTS += --simplifierx
+test-k-simplifierx: KORE_REPl_OPTS += --simplifierx
```

Comment:
Did you mean `KORE_REPL_OPTS`? (with capital "L")

Reply:
Good catch!

```suggestion
test-k-simplifierx: KORE_REPL_OPTS += --simplifierx
```